ingestion_server/ingestion_server/slack.py

```diff
@@ -68,3 +68,13 @@ def info(text: str, summary: str = None) -> None:
 
 def error(text: str, summary: str = None) -> None:
     _message(text, summary, level=Level.ERROR)
+
+
+def status(model: str, text: str) -> None:
```

Comment:
Given this is just for the ingestion server module I think `status` is a fine name :slightly_smiling_face: